cmd/influxd/server_integration_test.go

```diff
@@ -1462,9 +1462,13 @@ func TestSingleServerDiags(t *testing.T) {
 	config.Monitoring.Enabled = true
 	config.Monitoring.WriteInterval = main.Duration(100 * time.Millisecond)
 	nodes := createCombinedNodeCluster(t, testName, dir, 1, config)
+	createDatabase(t, testName, nodes, "mydb")
```

Comment:
because we need to to actually generate some shard data for the diag module to process it. Before this change the server diagnostic was just processing some basic information but not shard data